sites/fast-website/src/public/style.css

```diff
@@ -65,4 +54,21 @@ body {
   font-size: 46px;
   line-height: 56px;
   margin: 0;
-}
\ No newline at end of file
+}
+
+.divider {
+  grid-column: 2;
+  margin-top: 100px;
+  margin-bottom: 60px;
+}
+
+.footer {
+  grid-column: 2;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 84px;
+}
+
+.footer-navigation-item {
+  margin-right: 50px;

```

Comment:
does this have any rtl issue? do you care about rtl in this site and do you have any ability to get the direction? I'm not sure of any of these, just asking

Reply:
margin-inline-end will swap this value in RTL if that is desired.